<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>StartupScreen</class>
  <widget class="QWidget" name="StartupScreen">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Startup Screen</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="RecipeCardWidget" name="recipeCard1">

        </widget>
      </item>
      <item>
        <widget class="RecipeCardWidget" name="recipeCard2">

        </widget>
      </item>
      <item>
        <widget class="RecipeCardWidget" name="recipeCard3">

        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
      <item>
        <widget class="QPushButton" name="learnRecipeButton">
          <property name="text">
            <string>Learn Recipe</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
